test/extended/testdata/test-imagesource-build.yaml

```diff
@@ -89,7 +89,7 @@ items:
           deploymentconfig: imagesourceapp
       spec:
         containers:
-        - image: imagesourceapp
+        - image: " "
```

Comment:
more is good, overlapping existing tests and in a place where no one who understands the thing that's being tested is responsible for owning the test case is maybe not so good.